isaric/parsers/uganda-vri-v1.toml

```diff
@@ -155,6 +155,11 @@
   field = "outcome_date"
   if = { outcome = 2 }
 
+[subject.vaccinated_covid19]
+  field = "vcard"
```

Comment:
the variable vcard relates to the review of childhood vaccinations for DPT-HBV-HIB and doesn't indicate covid vaccination. I don't think there are any variables in this dataset that can be mapped here